app/models/transfer_request.rb

```diff
@@ -17,7 +17,7 @@ class TransferRequest < ApplicationRecord # rubocop:todo Metrics/ClassLength
 
   # Determines if we attempt to filter out {Aliquot#equivalent? equivalent} aliquots
   # before performing transfers.
-  attr_accessor :merge_equivalent_aliquots
+  attr_accessor :merge_equivalent_aliquots, :tag_depth
```

Comment:
I'm not sure we want to expose this as a separate attribute on transfer request, when https://github.com/sanger/sequencescape/pull/3338 is about to expose 'aliquot_attributes'. The advantage of the latter is we can potentially set any of the attributes on aliquot easily, without needing to modify transfer request. This approach would also let us set, say, the suboptimal flag, on an aliquot automatically when creating a descendant. (Something that may be needed for cardinal)

So instead of 
```
{
    "transfer_request_collection":{
      "user": "#{user.uuid}",
      "transfer_requests": [
        {
          "source_asset":"#{asset.uuid}",
          "target_asset": "#{target_asset.uuid}",
          "submission": "#{submission.uuid}",
          "volume": 10,
          "tag_depth": 2
        }
      ]
    }
  }
```

You'd post:

```
{
    "transfer_request_collection":{
      "user": "#{user.uuid}",
      "transfer_requests": [
        {
          "source_asset":"#{asset.uuid}",
          "target_asset": "#{target_asset.uuid}",
          "submission": "#{submission.uuid}",
          "volume": 10,
          "aliquot_attributes": {
            "tag_depth": 2
          }
        }
      ]
    }
  }
```